src/appleseed-max-impl/appleseedrenderer/appleseedrenderer.cpp

```diff
@@ -240,6 +248,8 @@ int AppleseedRenderer::Open(
     int                     default_light_count,
     RendProgressCallback*   progress_cb)
 {
+    BroadcastNotification(NOTIFY_PRE_RENDER, static_cast<void*>(&rend_params));
```

Comment:
The cast is redundant: in C++ all pointers can be implicitly converted to `void*`.